resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -63,64 +63,105 @@ Item
         // the cloud contents area.
         Column
         {
-            anchors.centerIn: parent
+            anchors.horizontalCenter: parent.horizontalCenter
             width: parent.width
             height: childrenRect.height
 
             spacing: 20 * screenScaleFactor
 
-            Image  // Cloud image
+            // Cloud image
+            Image
             {
                 id: cloudImage
                 anchors.horizontalCenter: parent.horizontalCenter
                 source: UM.Theme.getImage("first_run_ultimaker_cloud")
+                scale: 0.666
```

Comment:
This makes the size of this image depend on the internal coordinates of the SVG. For maintainability it would be better to explicitly give this a width or a height.
You also need to make this width or height depend on the screenScaleFactor so that the image is still in the same proportion as the rest of the interface if people use 200% screen scaling on high DPI screens or something.

The theme sizes automatically scale with the screenScaleFactor. Anything hard-coded, such as the spacing on line 70, needs to be multiplied by `screenScaleFactor`.

I realise that this also used to be an issue before your changes. It would be good to fix it anyway though now that we discover it!